ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1994,3 +1994,17 @@ def get_logsize(host, logfile):
     """ get current logsize"""
     logsize = len(host.get_file_contents(logfile))
     return logsize
+
+
+def pkg_installer(host):
+    """ Package installer for RHEL/Fedora and Ubuntu"""
+    cmd = 'cat /etc/os-release |head -n 1| cut -d "=" -f2'
```

Comment:
@menonsudhir is adding similar code to install packages in PR https://github.com/freeipa/freeipa/pull/4108, you should sync with him and leverage his code.

Reply:
@flo-renaud, thanks will reuse that